diff --git a/LunarisDolby/Android.bp b/LunarisDolby/Android.bp
index cd12a9f..711f3c4 100644
--- a/LunarisDolby/Android.bp
+++ b/LunarisDolby/Android.bp
@@ -35,7 +35,7 @@ android_app {
     ],
 
     required: [
-        "preinstalled-packages-platform-dolby.xml",
+        "preinstalled-packages-platform-lunarisdolby.xml",
         "privapp-permissions-dolby.xml",
     ],
 
@@ -45,7 +45,7 @@ android_app {
 }
 
 prebuilt_etc {
-    name: "preinstalled-packages-platform-dolby.xml",
+    name: "preinstalled-packages-platform-lunarisdolby.xml",
     src: "preinstalled-packages-platform-dolby.xml",
     sub_dir: "sysconfig",
     system_ext_specific: true,
